docs-api/frontity-packages/features-packages/wp-comments.md

```diff
@@ -0,0 +1,261 @@
+  - [In Frontity](#in-frontity)
+- [Usage](#usage)
+  - [Getting comments of a post](#getting-comments-of-a-post)
+  - [Sending new comments for a post](#sending-new-comments-for-a-post)
+- [API Reference](#api-reference)
+  - [Handlers](#handlers)
+    - [`@comments/:id`](#commentsid)
+  - [State](#state)
+    - [`state.comments.forms[]`](#state-comments-forms)
+    - [`state.source.comment[id]`](#state-source-comment-id)
+  - [Actions](#actions)
+    - [`actions.comments.updateFields()`](#actions-comments-updatefields)
+    - [`actions.comments.submit()`](#actions-comments-submit)
+- [Demo](#demo)
+
+<!-- tocstop -->
+
+## Installation
+
+Add the `wp-comments` package to your project:
+
+```bash
+npm i @frontity/wp-comments
+```
+
+## Settings
+
+### In WordPress
+
+In order to use this package, you will need to add a single line of configuration to your Wordpress installation:
+
+```php
+add_filter( 'rest_allow_anonymous_comments', '__return_true' );
+```
+
+[This filter](https://developer.wordpress.org/reference/hooks/rest_allow_anonymous_comments/) enables creating comments for anonymous users via the REST API.
+
+You can add this that directly in your theme's `functions.php` file or use a [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin.
+
+### In Frontity
+
+This package doesn't have any configuration. It just needs to be added to the `packages` array in `frontity.settings.js`.
+
+**`frontity.settings.js`**
+
+```js
+export default {
+  packages: ["@frontity/wp-comments"],
+};
+```
+
+## Usage
+
+### Getting comments of a post
+
+We can use the `@comments/:id` handler to fetch all the comments of a specific post (`actions.source.fetch("@comments/60")`). This data will be populated to the state so then we can do `state.source.get("@comments/60")` to get the ID's of these comments. With each ID we can get the details from the state at `state.source.comment[id]`
+
+{% hint style="info" %}
+Have a look at the "Getting Comments via the handler`@comments/:id`" diagram available in [excalidraw](https://excalidraw.com/#json=5946337115766784,uDV5oMBePTHTX8ppRwDt5g)
+{% endhint %}
+
+### Sending new comments for a post
+
+Every post with a comments' form (to send comments) will use `state.comments.forms[postId]` to store the data of the comment and the submission status
+
+The data at `state.comments.forms[postId]` can be updated through the action `actions.comments.updateFields()`
+
+To send new comments you can use the action `actions.comments.submit()` which will send the data available at `state.comments.forms[postId].fields`
+
+The submission status will be stored under under `state.comments.forms[postId]` and if there are errors they will be available at the properties `errorMessage`, `errorCode` and `errorStatusCode`
+
+{% hint style="info" %}
+Have a look at the "Sending new Comments via a React form" diagram available in [excalidraw](https://excalidraw.com/#json=5946337115766784,uDV5oMBePTHTX8ppRwDt5g)
+{% endhint %}
```

Comment:
This links to the same "Getting comments..." image as above. It doesn't link to a "Sending new comments..." image.